src/components/common/messaging/attachments/Attachment.tsx

```diff
@@ -24,6 +24,8 @@ export default function Attachment({ attachment, hasContent }: Props) {
     const { openScreen } = useIntermediate();
     const { filename, metadata } = attachment;
     const [spoiler, setSpoiler] = useState(filename.startsWith("SPOILER_"));
+    // only used in image attachments
+    const [loading, setLoading] = useState(true);
```

Comment:
As I stated in the PR description, this currently feels hacky. I'm thinking of spinning off image attachments as its own component (i.e. `ImageFile.tsx`). Such a change feels drastic, so I'd like to ask first, if it's fine to do that, or if there are other, better solutions.

Reply:
That would be fine.